sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/spark_job_entry_mixin.py

```diff
@@ -39,6 +39,11 @@ def entry(self, value: Union[Dict[str, str], SparkJobEntry, None]):
         self._entry = value
 
     def _validate_entry(self):
+        if self.entry is None:
```

Comment:
shall we change the msg in below if branch?

Reply:
updated